app/models/dns-record.server.ts

```diff
@@ -112,14 +112,25 @@ export async function doesDnsRecordExist(
   data: Pick<DnsRecord, 'username' | 'type' | 'subdomain' | 'value'>
 ) {
   const { username, type, subdomain, value } = data;
-  const count = await prisma.dnsRecord.count({
-    where: {
-      username,
-      type,
-      subdomain,
-      value,
-    },
-  });
+  let count = 0;
+  if (type === 'CNAME') {
```

Comment:
I think we can achieve same with this code
```js
const count = await prisma.dnsRecord.count({
    where: {
      username,
      type,
      subdomain,
      value: type === 'CNAME' ? undefined : value,
    },
  });
```

Reply:
Good idea!